website/pages/docs/reference/manifest.mdx

```diff
@@ -40,6 +40,19 @@ Scarb bakes in the concept of [Semantic Versioning](https://semver.org/), so mak
    Always increment the minor version if you add any new public structs, traits, fields, types, functions, methods, impls or anything else.
 4. Use version numbers with three numeric parts such as 1.0.0 rather than 1.0.
 
+### `cairo-version`
+
+The `cairo-version` field is an optional key that tells Scarb what version of the Cairo language and compiler your package can be compiled with.
+If the currently selected version of the Scarb compiler is older than the stated version, Scarb will exit with an error, telling the user what version is required.
```

Comment:
> compiler is older
Does it necessarily need to be older? It seems to me that something like "does not match requirements" would be more precise.

Reply:
Done.